packages/devtools_app/lib/src/logging/config.dart

```diff
@@ -0,0 +1,5 @@
+const String platformLogTag = 'platformLog';
```

Comment:
rather than adding arbitrary string-value pairs to a Map, can you add this field directly to the controller?
You can use ValueListenable class to make the way the state is tracked easy to port to Flutter in the future.